src/locales/ko/bgm-name.json

```
@@ -147,8 +147,8 @@
   "menu": "불가사의 던전 하늘의 탐험대 포켓몬 세계에 온 것을 환영한다!",
   "title": "불가사의 던전 하늘의 탐험대 메뉴 테마",
 
-  "mystery_encounter_weird_dream": "PMD EoS Temporal Spire",
-  "mystery_encounter_fun_and_games": "PMD EoS Guildmaster Wigglytuff",
-  "mystery_encounter_gen_5_gts": "BW GTS",
-  "mystery_encounter_gen_6_gts": "XY GTS"
+  "mystery_encounter_weird_dream": "PMD EoS Temporal Spire.",
+  "mystery_encounter_fun_and_games": "PMD EoS Guildmaster Wigglytuff.",
+  "mystery_encounter_gen_5_gts": "BW GTS.",
+  "mystery_encounter_gen_6_gts": "XY GTS."
```

```suggestion
  "mystery_encounter_weird_dream": "PMD EoS Temporal Spire",
  "mystery_encounter_fun_and_games": "PMD EoS Guildmaster Wigglytuff",
  "mystery_encounter_gen_5_gts": "BW GTS",
  "mystery_encounter_gen_6_gts": "XY GTS"
```
이미 미번역 배경음악이 좀 있고 우선순위가 크진 않아서 ME 이후에 몰아서 처리하죠